orchestrator/gorc/src/commands/sign_delegate_keys.rs

```diff
@@ -1,43 +1,53 @@
 use crate::application::APP;
 use abscissa_core::{Application, Command, Options, Runnable};
+use cosmos_gravity::query as QueryValSet;
```

Comment:
I would remove the ‘as QueryValSet’ part of this line, so later, you can call the ‘query::query_valset’ function more clearly.